lib/config/options/index.ts

```diff
@@ -1197,6 +1197,13 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'autoReplaceRegexGlobalMatch',
+    description:
+      'Use to control whether replacement regular expressions are global matches or only the first match.',
```

Comment:
```suggestion
      'Control whether replacement regular expressions are global matches or only the first match.',
```

Looks like we can simplify a bit?